src/Tizen.Multimedia.Remoting/MediaController/MediaControlMetadata.cs

```diff
@@ -116,5 +125,150 @@ internal MediaControlMetadata(IntPtr handle)
+        /// <since_tizen> 6 </since_tizen>
+        public Size Resolution
+        {
+            get => DecodeResolution(EncodedResolution);
+            set => EncodedResolution = EncodeResolution(value.Width, value.Height);
+        }
+
+        // Native CAPI used only encoded string to gets or sets Season, Episode, Resolution.
+        // But encoded string is not useful for user, so we don't offer as it is.
+        // It'll be used internally.
+        internal string EncodedSeason { get; private set; }
+
+        internal string EncodedEpisode { get; private set; }
+
+        internal string EncodedResolution { get; private set; }
+
+        private string EncodeSeason(SeriesInformation information)
+        {
+            Native.EncodeSeason(information.Number, information.Title, out string encodedSeason).
+                ThrowIfError("Failed to encode season");
+
+            return encodedSeason;
+        }
+
+        private string EncodeEpisode(SeriesInformation information)
+        {
+            Native.EncodeEpisode(information.Number, information.Title, out string encodedEpisode).
+                ThrowIfError("Failed to encode episode");
+
+            return encodedEpisode;
+        }
+
+        private string EncodeResolution(int width, int height)
+        {
+            Native.EncodeResolution((uint)width, (uint)height, out string encodedResolution).
+                ThrowIfError("Failed to encode resolution");
+
+            return encodedResolution;
+        }
+
+        private SeriesInformation DecodeSeason(string encodedSeason)
+        {
+            int number = 0;
+            string title = null;
+
+            if (encodedSeason != null)
+            {
+                Native.DecodeSeason(encodedSeason, out number, out title).
+                    ThrowIfError("Failed to decode season");
+            }
+
+            return new SeriesInformation(number, title);
+        }
+
+        private SeriesInformation DecodeEpisode(string encodedEpisode)
+        {
+            int number = 0;
+            string title = null;
+
+            if (encodedEpisode != null)
+            {
+                Native.DecodeEpisode(encodedEpisode, out number, out title).
+                    ThrowIfError("Failed to decode episode");
+            }
+
+            return new SeriesInformation(number, title);
+        }
+
+        private Size DecodeResolution(string encodedResolution)
+        {
+            uint width = 0, height = 0;
+
+            if (encodedResolution != null)
+            {
+                Native.DecodeResolution(encodedResolution, out width, out height).
+                    ThrowIfError("Failed to decode resolution");
+            }
+
+            return new Size((int)width, (int)height);
+        }
+    }
+
+    /// <summary>
+    /// Represents properties for the video series information.
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public struct SeriesInformation
```

Comment:
Do you have any reason to use `struct` instead of `class`?

Reply:
It is just for returning values so I chose value type. It doesn't need to inherit something.
Is there any reason to use class in this case?

Reply:
Because, usually use class, 

https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/choosing-between-class-and-struct